RecoLocalTracker/SiPixelClusterizer/plugins/gpuCalibPixel.h

```diff
@@ -4,14 +4,15 @@
 #include <cstdint>
 #include <cstdio>
 
+#include "CUDADataFormats/SiPixelCluster/interface/gpuClusteringConstants.h"
 #include "CondFormats/SiPixelObjects/interface/SiPixelGainForHLTonGPU.h"
 #include "HeterogeneousCore/CUDAUtilities/interface/cuda_assert.h"
 
 #include "gpuClusteringConstants.h"
```

Comment:
twice? or are there two `gpuClusteringConstants` files?

Reply:
Right...
`RecoLocalTracker/SiPixelClusterizer/plugins/gpuClusteringConstants.h` simply includes `CUDADataFormats/SiPixelCluster/interface/gpuClusteringConstants.h`, and should be removed.